src/app/zones/locales/en-us/index.yaml

```diff
@@ -1,30 +1,48 @@
 zone-cps:
   routes:
     item:
-      title: "{name} Zone Control Plane"
+      title: '{name} Zone Control Plane'
```

Comment:
nitpick: just to minimise changes like this, can we easily have a linter to enforce this?

Reply:
You’re right, we should lint this. FWIW the inconsistencies in our YAML file (aside from the casing/naming question) are fairly minimal so we aren’t hard-pressed to do so soon.